### PR TITLE
dash/pm-demo-css

### PR DESCRIPTION
**Pull request by karolkolodziej**
Changed CSS path.

### CODE REVIEW

**Review by sebastianbochan — APPROVED** [review]

File: samples/dashboards/demo/project-management/demo.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @import url("https://code.highcharts.com/css/highcharts.css");
 @import url("https://code.highcharts.com/css/datagrid.css");
```
Comment (goransle) [inline samples/dashboards/demo/project-management/demo.css:2]: Should probably do this one as well ( --> https://code.highcharts.com/datagrid/css/datagrid.css). https://code.highcharts.com/css/datagrid.css is still there, but seems to be an older version

Comment (sebastianbochan) [reply samples/dashboards/demo/project-management/demo.css:2]: I do not remember that we use  https://code.highcharts.com/datagrid/css/datagrid.css anywhere, should we?

Comment (goransle) [reply samples/dashboards/demo/project-management/demo.css:2]: Looks like https://code.highcharts.com/dashboards/css/datagrid.css is also still an option, but maybe /datagrid would be better for future-proofing? cc @oysteinmoseng

Comment (karolkolodziej) [reply samples/dashboards/demo/project-management/demo.css:2]: Good point but actually we don't use the DataGrid in this demo so it can be removed 😉

Comment (goransle) [reply samples/dashboards/demo/project-management/demo.css]: 😆

**Review by goransle — APPROVED** [review]

**Pull request merged**